src/style.css

```diff
@@ -1,3 +1,67 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  justify-content: center;
+}
+
+.nav_list {
```

Comment:
The class name `.nav_list` should be `.nav__list` to follow the BEM naming convention which represents the element part of the block `nav`.